sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/V2WriteAnalysisSuite.scala

```diff
@@ -238,12 +238,28 @@ abstract class V2WriteAnalysisSuiteBase extends AnalysisTest {
   def byPosition(table: NamedRelation, query: LogicalPlan): LogicalPlan
 
   test("SPARK-33136: output resolved on complex types for V2 write commands") {
```

Comment:
Thank you for pinging me.
If you don't mind, please add a new test case instead of changing the existing test case, @edgarRd .